wp-content/themes/citylimits/functions.php

```diff
@@ -606,3 +607,81 @@ function cftl_tax_landing_save_custom_fields( $post_id ){
 
 }
 add_action('save_post', 'cftl_tax_landing_save_custom_fields');
+
+/**
+ * Function to determine which partial slug should be used by LMP to render posts.
+ *
+ * Includes a "largo_lmp_template_partial" filter to allow for modifying the value $partial.
+ *
+ * @param WP_Query $post_query The WP_Query being used to generate LMP markup
+ * @return string $partial The slug of partial that should be loaded.
+ * @global $opt
+ * @global $_POST
+ * @see largo_load_more_posts
+ * @since 0.5.3
+ */
+function largo_load_more_posts_choose_partial($post_query) {
+	global $opt, $post;
+
+	$query_vars = $post_query->query_vars;
+
+	// Default is to use partials/content-home.php
+	$partial = 'home';
+	// This might be a category, tag, search, date, author, non-landing-page series, or other other archive
+	// check if this query is for a category
+	if ( isset($query_vars['category_name']) && $query_vars['category_name'] != '' ) {
+		$partial = 'archive';
+	}
+	// check if this query is for an author page
+	if ( isset($query_vars['author_name']) && $query_vars['author_name'] != '' ) {
+		$partial = 'archive';
+	}
+	// check if this query is for a tag
+	if ( isset($query_vars['tag']) && $query_vars['tag'] != '' ) {
+		$partial = 'archive';
+	}
+	// check if this query is for a search
+	if ( isset($query_vars['s']) && $query_vars['s'] != '' ) {
+		$partial = 'search';
+	}
+	// check if this query is for a date, assuming that all date queries have a year.
+	if ( isset($query_vars['year']) && $query_vars['year'] != 0 ) {
+		$partial = 'archive';
+	}
+	// Series landing pages
+	if ( isset($_POST['is_series_landing']) && $_POST['is_series_landing'] == 'true') {
+
+		if( true === $query_vars['is_series_featured_content_widget'] ) {
```

Comment:
```
if ( isset( $query_vars['is_series_featured_content_widget'] ) && true === $query_vars['is_series_featured_content_widget'] ) ) 
```

Reply:
@benlk https://github.com/INN/umbrella-citylimits/pull/100/commits/7226afc0d4f32bcbac0d0f2eafad9a990c981e76 updates that line and also puts it inside of the filter instead of redeclaring the function.